pytest.ini

```diff
@@ -0,0 +1,6 @@
+#pytest.ini
```

Comment:
this file should be the same as the one in the python template: https://github.com/OsaroAI/osaro-default-python-template/blob/main/pytest.ini